cmd/query/app/server.go

```diff
@@ -64,10 +60,16 @@ type Server struct {
 	httpServer    *httpServer
 	separatePorts bool
 	bgFinished    sync.WaitGroup
+	telset        telemetery.Setting
```

Comment:
how about embedding it? Then you wouldn't need to keep saying `s.telset.Logger`, simply `s.Logger` would work (even though it makes those fields public on the struct, but it's still better for readability than the long form `s.telset.*` everywhere).
```suggestion
	telemetery.Setting
```